### PR TITLE
Fix nfq-exp flag and nfq logging messages

### PR DESCRIPTION
**Pull request by dhaavi**

### CODE REVIEW

File: firewall/interception/nfqueue_linux.go [hunk 1]
```diff
@@ -36,10 +34,10 @@ var (
 )
 
 func init() {
-	flag.BoolVar(&experimentalNfqueueBackend, "experimental-nfqueue", true, "(deprecated flag; always used)")
+	flag.BoolVar(&experimentalNfqueueBackend, "experimental-nfqueue", false, "(deprecated flag; always used)")
```
Comment (ppacher) [inline firewall/interception/nfqueue_linux.go:37]: Why did you change that to false? It's always used that's what it defaults to `true`? (although, it doesn't matter because the flag isn't used anyway)

**Review by ppacher — APPROVED** [review]

File: firewall/interception/nfqueue_linux.go (hunk 1 above)
Comment (dhaavi) [reply firewall/interception/nfqueue_linux.go:37]: Because it then always shows the deprecated warnings.
